# 365 Probabilidades — Dia #007
## Qual a probabilidade de você estar feliz agora?

**Tipo:** Comportamento
**Data de publicação:** 2026-06-20
**Ferramenta:** Python
**Decisão analisada:** O que mais impacta a minha felicidade no dia a dia?
**Hashtag:** #365Probabilidades #Dia007

---

### 📖 A História

Tem um momento que quase todo mundo conhece. Você está fazendo uma coisa — lavando a louça, numa reunião, dirigindo — e percebe que a sua mente foi embora. Está pensando no que aconteceu ontem, no que pode acontecer amanhã, numa conversa que ainda não teve. O corpo está ali, a mente, não.

Pesquisadores de Harvard decidiram medir isso. Criaram um aplicativo que enviava notificações aleatórias para 15.000 pessoas ao longo do dia. A pergunta era simples: o que você está fazendo agora? Sua mente está focada nisso? Você está feliz?

O que eles encontraram mudou a forma como entendemos a felicidade. Não é tanto o que você faz que determina como você se sente. É onde a sua mente está enquanto você faz.

Quando a mente divaga — e ela divaga quase metade do tempo — a felicidade cai. Não importa se o pensamento é neutro ou até agradável. A mente fora do presente é, em média, uma mente menos feliz. Os pesquisadores chegaram a uma conclusão direta: *a mente humana é uma mente que vaga, e vagar faz as pessoas infelizes.*

O dado que mais chama atenção: 47% do tempo acordado, estamos pensando em outra coisa que não o que estamos fazendo. Quase metade da nossa vida consciente acontece no passado ou no futuro — raramente aqui.

---

### 📚 O Conceito

O estudo de Killingsworth e Gilbert (2010) utilizou *experience sampling* — coleta de dados em tempo real via smartphone — para capturar estados mentais e emocionais em condições naturalísticas. Diferente de pesquisas que dependem de memória retrospectiva, o método captura o que as pessoas realmente sentem no momento em que sentem.

A variável central é a **mind-wandering**: o estado em que os pensamentos se desconectam da tarefa atual. O modelo calcula a probabilidade de mind-wandering (p = 0.47) e o impacto dessa condição na felicidade reportada, usando uma escala contínua.

O fator de correção (×0.80) é aplicado para ajustar diferenças culturais e de contexto entre a amostra original (adultos americanos em condições de vida cotidiana) e uma população geral mais ampla.

---

### 🧮 O Modelo
**Fontes:**
- Killingsworth, M.A. & Gilbert, D.T. (2010) — *A Wandering Mind Is an Unhappy Mind* — Science, 330(6006) — N=15.000
- Zanesco et al. (2019) — *Mind-wandering and alterations to default mode network connectivity* — NeuroImage — N=2.400


In [1]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

plt.rcParams['font.family'] = 'serif'
plt.rcParams['axes.spines.top'] = False
plt.rcParams['axes.spines.right'] = False
plt.rcParams['axes.grid'] = True
plt.rcParams['grid.alpha'] = 0.3
plt.rcParams['figure.facecolor'] = 'white'
plt.rcParams['axes.facecolor'] = 'white'

print('✅ Bibliotecas carregadas')

✅ Bibliotecas carregadas


In [2]:
# --- DADOS DA LITERATURA ---
# Fonte: Killingsworth & Gilbert (2010) — Science — N=15.000
# Fonte: Zanesco et al. (2019) — NeuroImage — N=2.400

n_fonte = 15000          # tamanho amostral principal
p_mind_wandering = 0.47  # proporção do tempo com mente divagando
p_infeliz_mw     = 0.79  # prob de reportar menor felicidade quando mente divaga
p_presente       = 0.53  # proporção do tempo com mente presente
fator_correcao   = 0.80  # fator de ajuste cultural padrão do projeto

# Atividades com maior frequência de mind-wandering (%)
atividades = [
    'Descansando',
    'Trabalhando',
    'Usando computador',
    'Transporte',
    'Cuidados pessoais',
    'Fazendo refeição',
    'Conversando',
    'Ouvindo música',
    'Exercício',
    'Relação íntima',
]
freq_mw = [0.65, 0.50, 0.52, 0.50, 0.52, 0.41, 0.35, 0.40, 0.30, 0.10]

print('=' * 65)
print('  DADOS DA LITERATURA')
print('=' * 65)
print(f'  Proporção do tempo com mente divagando : {p_mind_wandering*100:.0f}%')
print(f'  Menor felicidade quando mente divaga   : {p_infeliz_mw*100:.0f}% dos casos')
print(f'  Proporção do tempo com mente presente  : {p_presente*100:.0f}%')
print(f'  Fator de correção aplicado             : ×{fator_correcao}')
print(f'  Amostra principal (Killingsworth 2010) : N={n_fonte:,}')
print('=' * 65)

  DADOS DA LITERATURA
  Proporção do tempo com mente divagando : 47%
  Menor felicidade quando mente divaga   : 79% dos casos
  Proporção do tempo com mente presente  : 53%
  Fator de correção aplicado             : ×0.8
  Amostra principal (Killingsworth 2010) : N=15,000


In [3]:
# --- O MODELO ---
# Distribuição Beta com IC 95% para mind-wandering

alpha = p_mind_wandering * n_fonte
beta  = (1 - p_mind_wandering) * n_fonte
dist  = stats.beta(alpha, beta)
ic    = dist.interval(0.95)

# Estimativa corrigida
p_corrigido = p_mind_wandering * fator_correcao

# Modelo para impacto na felicidade
alpha_f = p_infeliz_mw * n_fonte
beta_f  = (1 - p_infeliz_mw) * n_fonte
dist_f  = stats.beta(alpha_f, beta_f)
ic_f    = dist_f.interval(0.95)
p_infeliz_corrigido = p_infeliz_mw * fator_correcao

print('=' * 65)
print('  MODELO — RESULTADO')
print('=' * 65)
print(f'  Mind-wandering — estimativa central : {p_mind_wandering*100:.1f}%')
print(f'  Mind-wandering — IC 95%             : [{ic[0]*100:.1f}%, {ic[1]*100:.1f}%]')
print(f'  Mind-wandering — corrigido (×0.80)  : {p_corrigido*100:.1f}%')
print()
print(f'  Impacto felicidade — estimativa     : {p_infeliz_mw*100:.1f}%')
print(f'  Impacto felicidade — IC 95%         : [{ic_f[0]*100:.1f}%, {ic_f[1]*100:.1f}%]')
print(f'  Impacto felicidade — corrigido      : {p_infeliz_corrigido*100:.1f}%')
print('=' * 65)

  MODELO — RESULTADO
  Mind-wandering — estimativa central : 47.0%
  Mind-wandering — IC 95%             : [46.2%, 47.8%]
  Mind-wandering — corrigido (×0.80)  : 37.6%

  Impacto felicidade — estimativa     : 79.0%
  Impacto felicidade — IC 95%         : [78.3%, 79.6%]
  Impacto felicidade — corrigido      : 63.2%


In [4]:
# --- VISUALIZAÇÃO ---

# GRÁFICO 1 — Mind-wandering por atividade
fig1, ax1 = plt.subplots(figsize=(12, 8))

cores = ['#c0392b' if f >= 0.47 else '#2a8a82' for f in freq_mw]
bars = ax1.barh(atividades, [f * 100 for f in freq_mw],
                color=cores, alpha=0.85, height=0.6)

ax1.axvline(x=47, color='#c8a84b', linestyle='--', linewidth=2,
            label='Média geral: 47%')

for bar, val in zip(bars, freq_mw):
    ax1.text(bar.get_width() + 0.8, bar.get_y() + bar.get_height() / 2,
             f'{val*100:.0f}%', va='center', fontsize=11, color='#333333')

ax1.set_xlabel('% do tempo com mente divagando', fontsize=12)
ax1.set_title('Mind-wandering por atividade\nKillingsworth & Gilbert (2010) — N=15.000',
              fontsize=14, pad=15)
ax1.legend(fontsize=11)
ax1.set_xlim(0, 80)

plt.tight_layout()
plt.savefig('dia-007-grafico-01-atividades.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Gráfico 1 salvo!')


# GRÁFICO 2 — Impacto na felicidade
fig2, ax2 = plt.subplots(figsize=(12, 8))

categorias = ['Presente e\natividade agradável',
              'Divagando e\npensamento neutro',
              'Presente e\natividade neutra',
              'Divagando e\npensamento agradável',
              'Divagando e\npensamento desagradável']
felicidade = [0.0, -0.79, -0.42, -0.31, -2.34]  # desvios relativos ao estado base

cores2 = ['#2a8a82' if v >= 0 else '#c0392b' for v in felicidade]
ax2.barh(categorias, felicidade, color=cores2, alpha=0.85, height=0.5)

ax2.axvline(x=0, color='#333333', linewidth=1)
ax2.set_xlabel('Impacto na felicidade (escala relativa)', fontsize=12)
ax2.set_title('Como o estado mental afeta a felicidade\nKillingsworth & Gilbert (2010)',
              fontsize=14, pad=15)

for i, (cat, val) in enumerate(zip(categorias, felicidade)):
    offset = 0.05 if val >= 0 else -0.05
    ha = 'left' if val >= 0 else 'right'
    ax2.text(val + offset, i, f'{val:+.2f}', va='center', ha=ha,
             fontsize=11, color='#333333')

plt.tight_layout()
plt.savefig('dia-007-grafico-02-felicidade.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Gráfico 2 salvo!')


# GRÁFICO 3 — Distribuição Beta (mind-wandering)
fig3, ax3 = plt.subplots(figsize=(12, 8))

x = np.linspace(ic[0] * 0.999, ic[1] * 1.001, 1000)
y = dist.pdf(x)

ax3.plot(x * 100, y, color='#c0392b', linewidth=3)
ax3.fill_between(x * 100, y, alpha=0.2, color='#c0392b')
ax3.axvline(x=ic[0] * 100, color='#c8a84b', linestyle='--', linewidth=2,
            label=f'IC 95%: [{ic[0]*100:.1f}%, {ic[1]*100:.1f}%]')
ax3.axvline(x=ic[1] * 100, color='#c8a84b', linestyle='--', linewidth=2)
ax3.axvline(x=p_mind_wandering * 100, color='#c0392b', linestyle='-',
            linewidth=2, alpha=0.6, label=f'Estimativa central: {p_mind_wandering*100:.0f}%')
ax3.axvline(x=p_corrigido * 100, color='#2a8a82', linestyle='-.',
            linewidth=2, label=f'Corrigido (×0.80): {p_corrigido*100:.1f}%')

ax3.set_xlabel('Probabilidade de mind-wandering (%)', fontsize=12)
ax3.set_ylabel('Densidade', fontsize=12)
ax3.set_title('Distribuição Beta — Probabilidade de mind-wandering\nKillingsworth & Gilbert (2010) — N=15.000',
              fontsize=14, pad=15)
ax3.legend(fontsize=11)

plt.tight_layout()
plt.savefig('dia-007-grafico-03-bayesiano.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Gráfico 3 salvo!')

✅ Gráfico 1 salvo!
✅ Gráfico 2 salvo!
✅ Gráfico 3 salvo!


### 💡 O Insight

A probabilidade de a mente estar divagando em qualquer momento do dia é de **47%** — e quando isso acontece, em **79% dos casos** a pessoa reporta níveis menores de felicidade. Após o fator de correção, a estimativa conservadora para uma população geral é de **37,6%** do tempo em estado de mente divagante com impacto negativo na percepção de bem-estar.

O dado mais contraintuitivo: pensar em algo agradável enquanto a mente divaga ainda assim reduz a felicidade, comparado a estar presente em uma atividade neutra. O problema não é o conteúdo dos pensamentos — é a ausência do momento presente.

*O que seria diferente se você estivesse totalmente presente nos próximos 60 minutos?*

---

### ⚠️ Limitações do Modelo
- Amostra original composta por adultos americanos com smartphone — viés de seleção relevante
- Mind-wandering foi autorreportado; a percepção do próprio estado mental tem limitações
- Correlação não implica causalidade: pessoas menos felizes podem ter maior tendência a divagar, e não o contrário
- O fator de correção (×0.80) é uma aproximação conservadora, não derivado de dados brasileiros específicos

*A ciência é honesta sobre o que não sabe. O modelo também.*

---

### 📎 Links
- Substack: [link do post]
- Instagram: [link do post]

---
*365 Probabilidades · Decidindo com dados, um dia de cada vez.*